packages/lab/src/__tests__/__e2e__/number-input/NumberInput.cy.tsx

```diff
@@ -117,24 +131,25 @@ describe("Number Input", () => {
     cy.findByRole("spinbutton").should("have.value", "-10");
   });
 
-  it("decrements specified floating point `step` value when clicking decrement button", () => {
-    cy.mount(<Default decimalPlaces={2} defaultValue={0.0} step={0.01} />);
+  it("decrements specified floating point `step` value when clicking decrement button by calculating decimal scale", () => {
+    cy.mount(<Default defaultValue={0.0} step={0.01} />);
 
     cy.findByLabelText("decrement value").realClick();
     cy.findByRole("spinbutton").should("have.value", "-0.01");
   });
 
-  it("decrements specified `step` and `stepBlock` value when using keyboards", () => {
+  it("decrements specified `step` and `stepMultiplier` value when using keyboards", () => {
     cy.mount(
-      <Default defaultValue={10} step={10} stepBlock={100} min={-2000} />,
+      <Default defaultValue={10} step={10} stepMultiplier={10} min={-2000} />,
     );
 
     cy.findByRole("spinbutton").focus().realPress("ArrowDown");
-    cy.findByRole("spinbutton").should("have.value", "0").realPress("PageDown");
-    cy.findByRole("spinbutton")
-      .should("have.value", "-100")
-      .realPress(["Shift", "ArrowDown"]);
-    cy.findByRole("spinbutton").should("have.value", "-200").realPress("Home");
+    cy.findByRole("spinbutton").should("have.value", "0");
+    cy.findByRole("spinbutton").focus().realPress("PageDown");
```

Comment:
```suggestion
    cy.realPress("PageDown");
```
Do we need to focus the element each time?